manifests/noderesourcetopology/cluster-role.yaml

```diff
@@ -14,10 +14,10 @@ rules:
   verbs: ["*"]
 - apiGroups: [""]
   resources: ["nodes"]
-  verbs: ["get", "list", "patch"]
+  verbs: ["get", "list", "watch"]
 - apiGroups: [""]
   resources: ["pods"]
-  verbs: ["get","list","watch","update"]
+  verbs: ["get", "list", "watch", "update", "delete"]
```

Comment:
do we really need to delete pods?

Reply:
I borrowed it from the RBAC permissions that I saw in the repo here: https://github.com/kubernetes-sigs/scheduler-plugins/blob/6e58632f57f80274dc5e0f97df7f1d1907cc7c25/manifests/install/charts/as-a-second-scheduler/templates/rbac.yaml#L28 but can remove it.

Reply:
Removed the `delete` verb.